src/main/java/by/custom/utilcalculator/domain/tree/CommandTree.java

```diff
@@ -19,18 +21,24 @@ private CommandTree() throws UtilsborCommandTreeReadingException {
         treeRoot = HelperTree.buildTree();
     }
 
-    public static CommandTree getInstance() {
+    public static CommandTree init() {
         return TreeHolder.TREE_HOLDER;
     }
 
     public Map<StepsIndicator, Command> getFieldsToCommands() {
         return fieldsToCommands;
     }
 
-    public boolean validateCommand(final Command requestingCommand, final UserProgress userProgress) {
+    public boolean validateCommandFromNode(final Command requestingCommand, final UserProgress userProgress) {
         return isRequestingCommandAcceptable(requestingCommand, getNode(userProgress));
     }
 
+    public Step getNextStepFromNode(final UserProgress userProgress) {
+        System.out.println("current node: " + getNode(userProgress).getKey());
```

Comment:
Please, rid `println`

Reply:
I can rid it, but is it necessary for now? It does great help for me during testing the bot: I can see the current step and the next step in console: if I have an error and nodes don't work in a proper way, I can discover this error from console easily. 

It is not a first time you ask me to delete println - why is it important? Let's discuss.

Reply:
Let's discuss it on call.